creating-virtual-machines/virtio-win.adoc

```diff
@@ -1,53 +1,132 @@
+
+Other drivers can be installed after the successful windows installation.
+====
+
+
+How to install during Windows install?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To install drivers before the Windows starts its install, make sure you have
+virtio-win package attached to your VirtualMachine as SATA CD-ROM.
+In the Windows installation, choose advanced install and load driver.
+Then please navigate to loaded Virtio CD-ROM and install one of viostor or vioscsi,
+depending on whichever you have set up.
+
+Step by step screenshots: 
+
+image::../../_images/virtio_custom_install_0.png[Windows install, 600,400]
+image::../../_images/virtio_custom_install_1.png[Custom install, 600,400]
+image::../../_images/virtio_custom_install_2.png[Disk install, 600,400]
+image::../../_images/virtio_custom_install_3.png[Choose medium, 600,400]
+image::../../_images/virtio_custom_install_4.png[Choose driver, 600,400]
+image::../../_images/virtio_custom_install_5.png[Continue install, 600,400]
+
+
+How to install after Windows install?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After windows install, please go to https://support.microsoft.com/en-us/help/4026149/windows-open-device-manager[Device Manager].
+There you should see undetected devices in "available devices" section. You can install virtio drivers one by
+one going through this list.
+
+
+image::../../_images/virtio_driver_install_0.png[Unknown devices, 600,400]
+image::../../_images/virtio_driver_install_1.png[Select driver by id, 600,400]
+image::../../_images/virtio_driver_install_2.png[Install driver, 600,400]
+image::../../_images/virtio_driver_install_3.png[Driver installed, 600,400]
+
+For more details on how to choose a proper driver and how to install the driver, 
+please refer to the https://access.redhat.com/articles/2470791[Windows Guest Virtual Machines on Red Hat Enterprise Linux 7].
+
+
+How to obtain virtio drivers? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The virtio Windows drivers are distributed in a form of 
+https://kubevirt.io/user-guide/docs/latest/creating-virtual-machines/disks-and-volumes.html#containerDisk[containerDisk],
+which can be simply mounted to the VirtualMachine.
+The container image, containing the disk is located at: https://hub.docker.com/r/kubevirt/virtio-container-disk
+and the image be pulled as any other docker container: 
+
+[bash]
+----
+docker pull kubevirt/virtio-container-disk
+----
+
+However, pulling image manually is not required, it will be downloaded if not present 
+by Kubernetes when deploying VirtualMachine.
+
+
+Attaching to VirtualMachine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 KubeVirt distributes virtio drivers for Microsoft Windows in form if
-container disk. The package contains the virtio drivers as well as qemu
-guest agent. The disk was tested on Microsoft Windows Server 2012.
+container disk. The package contains the virtio drivers 
+The disk was tested on Microsoft Windows Server 2012.
 However, it should work on Microsoft Windows 7 and up.
```

Comment:
so it doesn't contain the agent?

drivers are tested independently and support any relevant windows version. you do not need to say "should work" or anything like that.